src/xcode/ENA/ENA/Source/Views/ExposureSubmission/ExposureSubmissionImageCardCell.swift

```diff
@@ -72,7 +76,9 @@ class ExposureSubmissionImageCardCell: UITableViewCell {
 			let attributedText = NSMutableAttributedString(attributedString: attributedDescription)
 			descriptionLabel.attributedText = attributedText
 		}
-		self.accessibilityIdentifier = accessibilityIdentifier
+
+		cardView.accessibilityLabel = "\(title)\n\n\(description)"
```

Comment:
Why the two new lines?

Reply:
Enforces a short break during voice over indicating the separation of the two strings.